tck/src/main/java/ee/jakarta/tck/data/framework/read/only/NaturalNumber.java

```diff
@@ -36,9 +36,15 @@ public enum NumberType {
     @jakarta.nosql.Column
     private Short numBitsRequired;
 
+    // Sorting on enum types is vendor-specific in Jakarta Data.
+    // Use numTypeOrdinal for sorting instead.
     @jakarta.nosql.Column
+    @jakarta.persistence.Enumerated(jakarta.persistence.EnumType.STRING)
     private NumberType numType; // enum of ONE | PRIME | COMPOSITE
 
+    @jakarta.nosql.Column
+    private int numTypeOrdinal; // ordinal value of numType
```

Comment:
@njr-11 you changed to String, but you still using this one here: numTypeOrdinal.

Shall we remove this one and update the tests to work with the enum as String?

Reply:
> @njr-11 you changed to String, but you still using this one here: numTypeOrdinal.
> 
> Shall we remove this one and update the tests to work with the enum as String?

To repeat what I said earlier, doing that "isn't a solution on its own and doesn't fix the problem. Without the other changes I made here, the test would remain broken for providers that are neither JPA nor NoSQL."